api/v1/operator/client/cilium_operator_client.go

```diff
@@ -0,0 +1,120 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
(Noting the commit title here, not the actual change)

Commit title says "operator apiserver" which is slightly confusing to me as "apiserver" usually refers to the K8s apiserver. Could you clarify the title or simply avoid using "apiserver"? Maybe you meant "the API served by the Operator"?

Reply:
Done